parquet/src/main/java/org/apache/iceberg/parquet/ParquetBloomRowGroupFilter.java

```diff
@@ -120,6 +124,9 @@ private boolean eval(
         return ROWS_MIGHT_MATCH;
       }
 
+      LOG.info(
+          "Bloom Filters are used. The following fields have bloom filters : {}",
```

Comment:
The case sensitivity seems off. I think we should use `Bloom filter` vs `bloom filter` or `Bloom Filter`. 
Also, would it be more informative to report the intersection of the filter and all bloom filter columns? We probably don't care about having Bloom filters for columns that are not referenced by the filter?

Reply:
Fixed case and also changed to log the intersection of the filter and all bloom filter columns.